code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -229,3 +218,45 @@
 	throwforce = 0
 	breakouttime = 0
 	cuffsound = 'sound/weapons/cablecuff.ogg'
+
+//////////////////////////////
+// MARK: CRAFTING
+//////////////////////////////
+/obj/item/restraints/handcuffs/cable/attackby(obj/item/I, mob/user as mob, params)
+	..()
+	// Don't allow borgs to send their their ziptie module to the shadow realm.
+	if(istype(src, /obj/item/restraints/handcuffs/cable/zipties/cyborg))
+		return
+
+	if(istype(I, /obj/item/stack/rods))
+		var/obj/item/stack/rods/R = I
+		if(!R.use(1))
+			to_chat(user, "<span class='warning'>[R.amount > 1 ? "These rods" : "This rod"] somehow can't be used for crafting!</span>")
+			return
+
+		var/obj/item/wirerod/W = new /obj/item/wirerod
+		if(user.unEquip(src) || !remove_item_from_storage(user))
+			user.put_in_hands(W)
+		to_chat(user, "<span class='notice'>You wrap the cable restraint around the top of the rod.</span>")
+		qdel(src)
+		return
+
+	if(istype(I, /obj/item/stack/sheet/metal))
+		var/obj/item/stack/sheet/metal/M = I
+		if(M.amount < 6)
+			to_chat(user, "<span class='warning'>You need at least six metal sheets to make good enough weights!</span>")
+			return
+
+		to_chat(user, "<span class='notice'>You begin to apply [I] to [src]...</span>")
+		if(M.use(6) && do_after(user, 35 * M.toolspeed, target = src))
+			var/obj/item/restraints/legcuffs/bola/S = new /obj/item/restraints/legcuffs/bola
```

Comment:
```suggestion
			var/obj/item/restraints/legcuffs/bola/S = new /obj/item/restraints/legcuffs/bola(get_turf(src))
```
Extra precaution in case `put_in_hands()` fails to do it's thing